app/scripts/actions/nexgddptool.js

```diff
@@ -297,9 +297,13 @@ export function setTempResolutionSelection(selection, changeUrl = true) {
 
     // We need to reset the date selectors too
     const range1Options = getState().nexgddptool.range1.options;
+    const currentYear = (new Date()).getUTCFullYear();
+    const selectedOption = range1Options[selection.value]
+      .find(option => currentYear > option.label.split('-')[0]
```

Comment:
I think that the best condition would be `>=` and `<=`. 
Example: you are in 2020 and you have this range of dates `['2001-2010', '2011-2020','2021-2030']`.

Reply:
Good catch! Thanks for the review.